internal/setup/setup_test.go

```diff
@@ -1309,6 +1309,28 @@ var setupTests = []setupTest{{
 		`,
 	},
 	// TODO this should be an error because the content does not match.
+}, {
+	summary: "chisel-v1 is deprecated",
+	input: map[string]string{
+		"chisel.yaml": `
+			format: chisel-v1
+			archives:
+				foo:
+					version: 22.04
+					components: [main, universe]
+					suites: [jammy]
+					v1-public-keys: [extra-key]
+					default: true
+			v1-public-keys:
+				extra-key:
```

Comment:
Does this key have any significance? `extra-key` seems to imply that it is different from a regular one.

Reply:
Removed it in https://github.com/letFunny/chisel/pull/16/commits/18e61a7dcf9490f62936fa6ae4a864ff68b19a30.

Reply:
Can you add the `v1-public-keys` back? I only meant using the default key instead of implying that this is an extra one. I think having a complete example of `chisel-v1` is still useful for reference.

Reply:
Sure.

Reply:
Sure. Added in b869141.